frontend/package.json

```diff
@@ -40,15 +40,16 @@
       ".(ts|tsx|js|jsx)": "./node_modules/ts-jest/preprocessor.js"
     },
     "transformIgnorePatterns": [
-      "<rootDir>/node_modules/(?!lodash-es|@console)"
+      "<rootDir>/node_modules/(?!(lodash-es|@console|@novnc|@spice-project)/.*)"
```

Comment:
The list of pkgs that require either ESnext or TS transpilation via `ts-jest` or `ts-node` keeps growing.

In a follow-up, I'd like to extract Jest config into a separate JS file and either try to reuse the same list of pkgs, or just un-ignore the whole `node_modules` content.

Reply:
#2057 should fix this.